### PR TITLE
vesktop: Add version 1.5.0

### PR DESCRIPTION
**Pull request by TayouVR**
this PR adds the [Vesktop App](https://github.com/Vencord/Vesktop) to this bucket

this json file is basically just mirrored from https://github.com/anderlli0053/DEV-tools/blob/master/bucket/vesktop.json and https://github.com/okibcn/ScoopMaster/blob/master/bucket/vesktop.json 
as I think Vesktop should be in one of the official repos; Extras being the most fitting.

Closes https://github.com/Vencord/Vesktop/issues/82

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/vesktop.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "github": "https://github.com/Vencord/Vesktop"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Vencord/Vesktop/releases/download/v$version/Vesktop-Setup-$version.exe#/dl.7z"
+            }
+        }
+    }
+}
```
Comment (ArjixWasTaken) [inline bucket/vesktop.json:35]: Vesktop now has a portable folder, w/o a setup, available.
This means we can avoid having an installer script, which makes this easier to maintain.

Here is my suggested manifest:
```suggestion
{
    "version":  "0.3.3",
    "description": "A cross platform electron-based desktop app aiming to give you a snappier Discord experience with Vencord pre-installed",
    "homepage":  "https://github.com/Vencord/Vesktop",
    "license":  "GPL-3.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/Vencord/Vesktop/releases/download/v0.3.3/Vesktop-0.3.3-win.zip",
            "hash": "faca0728ee67c31469f532412c30c5c21aa3aa41283111a62c210313acb5b2c3"
        }
    },
    "bin": "Vesktop.exe",
    "shortcuts":  [[
        "Vesktop.exe",
        "Vesktop"
    ]],
    "checkver": {
        "github": "https://github.com/Vencord/Vesktop"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Vencord/Vesktop/releases/download/v$version/Vesktop-$version-win.zip"
            }
        }
    }
}`

PS: It appears that your manifest downloads the archive as `dl.7z`, but you refer to it as `app-dl.7z` in your install script.
I am new to scoop manifests, but this looks like a mistake?

**Review by ArjixWasTaken — CHANGES_REQUESTED** [review]

File: bucket/vesktop.json (hunk 1 above)
Comment (ArjixWasTaken) [reply bucket/vesktop.json:35]: PS: It appears that your manifest downloads the archive as `dl.7z`, but you refer to it as `app-dl.7z` in your install script.
I am new to scoop manifests, but this looks like a mistake?

Comment (TayouVR) [reply bucket/vesktop.json:35]: I am new to scoop manifests myself, your suggestion looks good though and I agree that a portable zip works better for scoop. Though I would keep the name "Vencord Desktop" as shortcut as that is the older name of the app. Unless that causes some other issues I'm not aware of..

**Pull request merged**